### PR TITLE
Data xenium breast cancer

### LINKED ISSUE #50: data_xenium_breast_cancer
**Update metadata:**
https://docs.google.com/spreadsheets/d/1QCeAF4yQG4bhZSGPQwwVBj_XF7ADY_2mK5xivAIfHsc/edit#gid=1453488771

**Details of data(publication, GitHub):**
"https://www.10xgenomics.com/products/xenium-in-situ/preview-dataset-human-breast
PRocessed: https://squidpy.readthedocs.io/en/stable/notebooks/tutorials/tutorial_xenium.html"

### PR DESCRIPTION
**Pull request by heylf**

### CODE REVIEW

**Comment by heylf** [comment]
Wait till you merge. I still check the scripts.

File: data/xenium-breast-cancer/environment.yml
Comment (niklasmueboe) [inline data/xenium-breast-cancer/environment.yml:1]: can you give the file the same name as the python file

File: data/xenium-breast-cancer/xenium-breast-cancer.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+def gunzip_file(gzipped_file_path, output_file_path):
+    with gzip.open(gzipped_file_path, 'rb') as f_in, open(output_file_path, 'wb') as f_out:
+        shutil.copyfileobj(f_in, f_out)
+
+get_data(out_dir):
+    with tempfile.TemporaryDirectory() as temp_dir:
+
+    # download the folder 'temporary_file_host_spacehack'   
+    download_data('https://cf.10xgenomics.com/samples/xenium/1.0.1/Xenium_FFPE_Human_Breast_Cancer_Rep1/Xenium_FFPE_Human_Breast_Cancer_Rep1_outs.zip',
```
Comment (niklasmueboe) [inline data/xenium-breast-cancer/xenium-breast-cancer.py:44]: the indentation seems to be off?
Has this script been tested?

File: data/xenium-breast-cancer/xenium-breast-cancer.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    download_data('https://cf.10xgenomics.com/samples/xenium/1.0.1/Xenium_FFPE_Human_Breast_Cancer_Rep2/Xenium_FFPE_Human_Breast_Cancer_Rep2_outs.zip', 
+                  f'{temp_dir}/replicate1', True)
+    download_data('https://cf.10xgenomics.com/samples/xenium/1.0.1/Xenium_FFPE_Human_Breast_Cancer_Rep2/Xenium_FFPE_Human_Breast_Cancer_Rep2_he_image.tif', 
+                  f'{temp_dir}/replicate1', False)
+    download_data('https://cf.10xgenomics.com/samples/xenium/1.0.1/Xenium_FFPE_Human_Breast_Cancer_Rep2/Xenium_FFPE_Human_Breast_Cancer_Rep2_gene_panel.json', 
+                  f'{temp_dir}', False)
+    download_data('https://cdn.10xgenomics.com/raw/upload/v1695234604/Xenium%20Preview%20Data/Cell_Barcode_Type_Matrices.xlsx', 
+                  f'{temp_dir}', False)
+
+    shutil.move(f'{tmpdir}/Xenium_FFPE_Human_Breast_Cancer_Rep2_gene_panel.json', f'{out_dir}/experiment.json')
```
Comment (niklasmueboe) [inline data/xenium-breast-cancer/xenium-breast-cancer.py:57]: Not sure where the variable `tmpdir` is coming from

File: data/xenium-breast-cancer/xenium-breast-cancer.py
Comment (niklasmueboe) [inline data/xenium-breast-cancer/xenium-breast-cancer.py:1]: add a contribution

**Review by niklasmueboe — CHANGES_REQUESTED** [review]

**Review by niklasmueboe — APPROVED** [review]

**Review by niklasmueboe — CHANGES_REQUESTED** [review]
Yaml seems to be mssing now

**Comment by heylf** [comment]
Done. Sorry was a silly mistake. Should be fine now and ready to merge.

**Review by niklasmueboe — APPROVED** [review]

**Pull request merged**